MoviesTVSentiments/app/src/test/java/com/google/moviestvsentiments/service/web/NetworkBoundResourceTest.java

```diff
@@ -0,0 +1,170 @@
+package com.google.moviestvsentiments.service.web;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import com.google.moviestvsentiments.util.LiveDataTestUtil;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import retrofit2.Response;
+
+@RunWith(JUnitParamsRunner.class)
+public class NetworkBoundResourceTest {
+
+    private static final String LOCAL_VALUE = "Local Value";
+    private static final String SERVER_VALUE = "Server Value";
+
+    @Rule
+    public InstantTaskExecutorRule instantExecutorRule = new InstantTaskExecutorRule();
+
+    private Object[] invokesCorrectMethodsParameters() {
+        return new Object[] {
+            new Object[] {false, new boolean[] {false, true, true, false} },
```

Comment:
I am a bit confused here. When shouldFetch=false, why is the shouldFetch called?

Reply:
Perhaps I can rename the variables a bit better. The first parameter is the value returned by shouldFetch

Reply:
Ah I see. Thanks